### PR TITLE
Add basic functionality tests for privacy requests

### PR DESCRIPTION
**Pull request by katrinDY**

### CODE REVIEW

File: cypress/integration/basic-functionalities/privacy/Add-DPO-user.spec.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      // We check if the success toast appears
+      cy.get('.b-toast-success')
+    })
+
+    it('should be able to add password', () => {
+      cy.get('[data-test-id="card-user-password"]').within(() => {
+        cy.get('[data-test-id="input-new-password"]').type(dpoPassword, { force: true })
+        cy.get('[data-test-id="input-confirm-password"]').type(dpoPassword, { force: true })
+        cy.get('[data-test-id="button-submit"]').click({ force: true })
+        cy.wait(1000)
```
Comment (petergrlica) [inline cypress/integration/basic-functionalities/privacy/Add-DPO-user.spec.js]: I thought we are going away from the `wait()` functionality?

**Review by petergrlica — CHANGES_REQUESTED** [review]
There should be no cy.wait() functionality as it is not very consistent and stable, especially running on a container infrastructure.

File: cypress/integration/basic-functionalities/privacy/Add-DPO-user.spec.js (hunk 1 above)
Comment (katrinDY) [reply cypress/integration/basic-functionalities/privacy/Add-DPO-user.spec.js]: The test won't pass unless there's a wait()

Comment (petergrlica) [reply cypress/integration/basic-functionalities/privacy/Add-DPO-user.spec.js]: can you do it similarly like here: `cy.get('.nav-sidebar', { timeout: 10000 }).contains('Applications')` ?
See more:
https://stackoverflow.com/questions/60063879/how-to-wait-for-an-element-to-be-visible
https://docs.cypress.io/faq/questions/using-cypress-faq#How-can-I-make-Cypress-wait-until-something-is-visible-in-the-DOM

**Pull request merged**